pkg/clusterversion/clusterversion.go

```diff
@@ -3,17 +3,23 @@ package clusterversion
 import (
 	"context"
 	"fmt"
+	"slices"
+	"time"
 
+	"github.com/Masterminds/semver/v3"
 	"github.com/golang/glog"
 	"github.com/openshift-kni/eco-goinfra/pkg/clients"
 	"github.com/openshift-kni/eco-goinfra/pkg/msg"
 	v1 "github.com/openshift/api/config/v1"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	clusterVersionName = "version"
+	isTrue             = "True"
 )
 
 // Builder provides a struct for clusterversion object from the cluster and a clusterversion definition.
```

Comment:
Please add errorMsg to Builder you'll re-use it later in With* functions. Refer to example: https://github.com/openshift-kni/eco-goinfra/blob/4b536d23d57aedc397057a32d67fd2823296326e/pkg/sriov/network.go#L32

Reply:
Added, thanks